### DISCUSSION TITLE
After upgrade to Chia 1.1.7 hung at circle "connecting to wallet"

### DISCUSSION THREAD

**Discussion by RollingRoland** [Chia-Network/chia-blockchain#6790] [Support]
I am just farming chia now in the background of my Windows 7 HTPC. I thought I get around and upgrade to 1.1.7. After I start the Chia GUI it just hangs a the circle "connecting to wallet". I tried rolling back to 1.1.6 and remove and fresh install of 1.1.7. Still hangs at the same point. On my former plotting machine with the old 1.1.6. it still starts and connects as normal.

**Comment by loppefaaret** [comment]
you might have a corrupted wallet database, and need to remove it, to sync a fresh database from 0:
WALLET DATABASE RESET

Close down the Chia app, and open up a PowerShell window, and execute the below command:
```
del ~\.chia\mainnet\wallet\db\block*
```
Then try opening up the Chia app again

**Comment by BrandtH22** [comment]
@RollingRoland , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
